docs/modules/reference/pages/glossary.adoc

```diff
@@ -66,10 +86,17 @@ Kafka:: Clusterable, open-source message broker designed for high-throughput/low
 Kibana:: Kibana is a component of the Elastic Stack (comprised of Elasticsearch, Logstash, and Kibana).
 It provides visualization capabilities for data indexed in an Elasticsearch cluster.
 
-KSC Reports:: Key SNMP customized (KSC) reports provide a way to generate prefabricated graphical views of collected data. 
+KSC Reports:: Key SNMP customized (KSC) reports provide a way to generate prefabricated graphical views of collected data.
 They let you display data from different devices and sources (SNMP, ICMP, HTTP) on one page.
 (See xref:operation:admin/webui/opsboard/dashlet/ksc.adoc#ksc[KSC Reports].)
 
+Kubernetes:: Kubernetes is an open-source container orchestration system for automating software deployments, scaling, and management.
+Originally designed by Google, it is now maintained by the Cloud Native Computing Foundation.
+(See https://kubernetes.io/[Kubernetes].)
+
+Location:: The geographical location of a node that is being managed using OpenNMS.
+Nodes can be manually assigned a location (See xref:operation:reference/provisioning/handlers/dns.adoc[Location]), or the location can be determined using the GeoIP Provisioning Adapter (See xref:operation:reference/provisioning/adapters/geoip.adoc[GeoIP]).
```

Comment:
In the product, the term "location" is generally more associated with "monitoring locations" and Minions, which have a specific impact when set on requisitioned nodes. In the context of geolocation, the DNS handler isn't relevant here.

Reply:
Ah, okay. In that case, do you have a suggestion on how this should be reworded? And should we have two entries - one for the monitoring locations and one for geolocation?